packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -875,6 +839,87 @@ export const uploadTemplateToS3 = async (
   }
 };
 
+const createResourceObject = (resource: string, category: string) : {
+  category: string,
+  key: string
+} => ({
+  category: `${category}-${resource}`,
+  key: category + resource,
+});
+
+const getCategoryResources = (file, fileSystem, resourceDir) => {
+  const cloudFormationJsonPath = path.join(resourceDir, file);
+  if (!fileSystem.exists(cloudFormationJsonPath)) {
```

Comment:
`context.filesystem` is deprecated.

```suggestion
  if (!fs.exists(cloudFormationJsonPath)) {
```